.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+before_install:
+  - docker pull koalaman/shellcheck
+
+script:
+  - docker run -v $(pwd):/scripts koalaman/shellcheck /scripts/base16-manager
```

Comment:
`shellcheck` is officially supported by travis. Therefore, we don't need to install it manually.
Just run
`-  shellcheck base16-manager`

Reply:
I am aware of this, but am following the trend of containerizing everything. I prefer running ShellCheck from the official Docker image. :)